packages/manager/apps/pci-object-storage/src/utils/getDashboardTabs.ts

```diff
@@ -0,0 +1,37 @@
+type TTab = {
+  name: string;
+  title: string;
+  to: string;
+};
+
+type TStorageTabsProps = {
+  projectId: string;
+  storageId: string;
+  region: string;
+  t: (key: string) => string;
+};
+
+export const getDashboardTabs = ({
```

Comment:
This is not an utils function ... 
This looks more a loader function with redirect rules

Reply:
BTW misses unit tests